scripts/start/prepare_and_run.R

```diff
@@ -245,11 +245,11 @@ prepare <- function() {
   setwd(cfg$remind_folder)
 
   # Check configuration for consistency
-  cfg <- check_config(cfg, reference_file="config/default.cfg",
-                      settings_config = "config/settings_config.csv",
-                      extras = c("backup", "remind_folder", "pathToMagpieReport", "cm_nash_autoconverge_lastrun",
-                                 "gms$c_expname", "restart_subsequent_runs", "gms$c_GDPpcScen",
-                                 "gms$cm_CES_configuration", "gms$c_description"))
+#  cfg <- check_config(cfg, reference_file="config/default.cfg",
```

Comment:
This is outcommented because there is no default.cfg anymore against which the cfg object could be compared, right? How do we deal with this in future? What is the purpose of the consistency check? Do we need a replacement for this in future?

Reply:
right, the new default.cfg does not include all settings and therefore the check would not work. We will have to introduce a similar check for our new starting script structure in the end